src/__tests__/actions.spec.ts

```diff
@@ -330,6 +465,13 @@ describe('Code Editor Shortcuts: actions', () => {
       expect(cursor.line).toEqual(0);
     });
 
+    it('should delete to the end of the line', () => {
+      deleteToEndOfLine(editor as any);
+
+      const { doc } = getDocumentAndSelection(editor);
+      expect(doc).toEqual('lorem ipsum\ndolor\namet');
+    });
```

Comment:
Is this a duplicate of https://github.com/RohanM/obsidian-emacs-shortcuts/blob/add-emacs-shortcuts/src/__tests__/actions.spec.ts#L116-L122?

Reply:
This spec starts with a range selection, so slightly different starting conditions.

Reply:
Ah of course! Thanks for clarifying 🙂